src/js/src/components/SearchFacetOptions.vue

```diff
@@ -11,9 +11,11 @@
         </div> 
         <div v-for="(facet, facetIndex) in facetCategory[1]"
              :key="facetIndex"
-             :class="facetIndex % 2 === 0 ? 'facetItem' : 'facetCount'"
-             @click="facetIndex % 2 === 0 ? applyFacet(facetCategory[0], facet) : null">
-          {{ facetIndex % 2 === 0 ? facet || "Unknown" : "(" + facet.toLocaleString("en") + ")" }}
+             :class="facetIndex % 2 === 0 ? 'facetItem' : 'facetCount'">
+          <router-link v-if="facetIndex % 2 === 0" :to="{ path: getFacetSelectionLink(facetCategory, facet) }">
+            {{ facet || 'Unknown' }}
+          </router-link>
```

Comment:
Of course - should have caught this in my first review - sorry.

Reply:
:+1: No worries - I should have realised we wanted links instead of relying on normal buttons instead. Its okey now tho :-)